src/model.whiley

```diff
@@ -8,85 +8,117 @@ public type State is {
     bool[] cells,
     uint width,
     uint height
-} where |cells| == width * height
+} where |cells| == width * height && width > 0 && height > 0
 
 /**
  * Intialise the game in a window with given dimensions.  Since the
  * width and height come from the Canvas properties, assume they are
  * non-negative.
+ *
  */
-public function init(uint width, uint height) -> State:
-    //
-    width = width / 20
-    height = height / 20
-    //
+public function init(uint width, uint height) -> (State r)
+requires width > 0 && height > 0
```

Comment:
Hey Liam,

Notice that you don't need `requires width > 0` since this is implicit in the type `uint`.